framework/xds_k8s_testcase.py

```diff
@@ -386,6 +388,40 @@ def removeServerBackends(self, *, server_runner=None):
         # Remove backends from the Backend Service
         self.td.backend_service_remove_neg_backends(neg_name, neg_zones)
 
+    def assertHealthyEndpointsCount(
```

Comment:
Please place the method just before `assertDrainingEndpointsCount` since they're so similar